person.rb

```diff
@@ -12,6 +12,7 @@ def initialize(age, name = 'unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
```

Comment:
- [ ] Kindly add the `attr_accessor` for this `rentals` at the top of the class.

  ```
  attr_accessor :rentals
  ```